### DISCUSSION TITLE
举报处

### DISCUSSION THREAD

**Discussion by share121** [share121/inter-knot#4497] [公告]
![微信图片_20240908100044](https://github.com/user-attachments/assets/1589cb41-5352-4718-b70c-2fe2e6f39445)

可以在这里举报

**Comment by Guessme404** [comment]
原来置顶的帖子怎么都没了

**Comment by hk0852** [comment]
涉及社区搬石，为了不让看到这帖子的人臭味开始在这个小社区里玩抽象所以举报

**Comment by Astrocapbled-Yeiro** [comment]
违规文章：#4561
举报原因：史

**Reply by 0xSelenicDove** [reply]
<img src="https://bbs-static.miyoushe.com/static/2023/05/26/00e65981c29461c8f04bdbd2c5dc3fe9_6783383035141785816.png" alt="胡桃-六！六！六！" width="75" >

**Comment by 927484059** [comment]
违规文章：#4668
举报原因：传播开盒

**Comment by ysdtal** [comment]
违规文章：#4668
举报原因：涉及开盒等敏感内容

**Comment by LMA936** [comment]
违规文章：https://github.com/share121/inter-knot/discussions/4668
举报原因：涉及开盒等敏感内容

**Comment by hk0852** [comment]
违规文章：#4668
举报原因：涉及开盒暴露他人的个人隐私信息

**Comment by ID104772508** [comment]
违规文章：#4714
举报原因：广告

**Comment by HyperMuteki0347** [comment]
违规文章：#4714
举报原因：广告

**Comment by ST-Diamond** [comment]
违规文章：#4714
举报原因：广告

**Comment by fuqikilroy** [comment]
违规文章：#4565
举报原因：低俗

**Comment by Junjun280** [comment]
违规文章：#4531
举报原因：试图发布修改绳网APP教程

**Comment by Astrocapbled-Yeiro** [comment]
违规文章：#4838
举报原因：低俗不良信息

**Comment by BJ-cmyk** [comment]
违规文章：#4668
举报原因：盒

**Comment by JiuXiat** [comment]
违规文章：#4820
举报原因：带节奏

**Comment by JiuXiat** [comment]
违规文章：#4828
举报原因：带节奏

**Comment by canifish** [comment]
违规文章：#4820
举报原因：带节奏

**Comment by HLXS-HLXS** [comment]
违规文章：https://github.com/share121/inter-knot/discussions/4820
举报原因：带节奏

**Comment by JulyLin555** [comment]
违规文章：#4668
举报原因：恶意开盒并传播

**Comment by BJ-cmyk** [comment]
违规文章：#4828
举报原因：节奏

**Comment by BJ-cmyk** [comment]
违规文章：#4668
举报原因：盒

**Comment by JulyLin555** [comment]
违规文章：#4820
举报原因：故意带节奏

**Comment by Liruijia666** [comment]
举报这个帖子  
理由：没有置顶

**Comment by Liruijia666** [comment]
违规讨论：#4565
举报原因：发帖人头像过于逆天

**Comment by March-777th** [comment]
违规讨论：#4668
举报原因：我超，盒！<img src="https://img-static.mihoyo.com/communityweb/upload/7bd889870907d2d96673d09afb53cf63.png" alt="三月七_哭" width="75" >

**Comment by Uknown-wang** [comment]
违规讨论：#4565
举报原因：传播违禁信息

**Comment by passbyDFX** [comment]
0.o

**Comment by hk0852** [comment]
违规讨论：#4874 
举报原因：发布节奏向内容

**Comment by BJ-cmyk** [comment]
违规讨论：#4874
举报原因：不干净的东西

**Comment by JiuXiat** [comment]
违规文章：#4874
举报原因：带节奏

**Comment by LV0Incapableperson** [comment]
违规讨论：#4874
举报原因：节奏

**Comment by duoduo13123678** [comment]
违规讨论：#4668
举报原因：侵犯他人肖像权

**Comment by KELERSAMA** [comment]
违规讨论：#4874
举报原因：节奏内容

**Comment by JiuXiat** [comment]
违规文章：#5041
举报原因：涩涩

**Comment by JT9364** [comment]
违规讨论：#4874
举报原因：节奏内容

**Comment by cuijiaqi0406** [comment]
违规讨论：#4874
举报原因：史，带节奏

**Comment by fzdqz** [comment]
#4874 删了吧，已经收集完绳匠的态度了

**Comment by Astrocapbled-Yeiro** [comment]
违规讨论：#5123
举报原因：传播不良信息（不要在这种地方发jm喂！）

**Comment by swyh750810** [comment]
违规讨论：#5123
举报原因：涉黄

**Comment by LV0Incapableperson** [comment]
违规讨论：#5123
举报原因：传播色情链接

**Comment by faatong** [comment]
违规讨论：#5123
举报原因：

**Comment by keyuanya** [comment]
举报讨论：#4874
举报原因：引战

**Comment by JiuXiat** [comment]
违规讨论：#5123
举报原因：公共场合传播色情内容

**Comment by mihayou123** [comment]
违规讨论：#5123
举报原因：传播色情内容

**Comment by JiuXiat** [comment]


**Reply by JiuXiat** [reply]
这里写错了，原本是另外一个帖子的，这应该是节奏向性内容

**Comment by keyuanya** [comment]
举报讨论：#5110
举报原因：引战

**Comment by TNCNET** [comment]
违规讨论：#5163
举报原因：传播NSFW/色情内容

**Comment by Junjun280** [comment]
违规讨论：#5120
举报原因：引战

**Comment by ParanoidHikari** [comment]
违规讨论：#5163
举报原因：传播色情内容

**Comment by Liruijia666** [comment]
违规讨论：#3028
举报原因：传播色情

**Comment by March-777th** [comment]
违规讨论：#5236
举报原因：盒

**Comment by JiuXiat** [comment]
违规讨论：#5236
举报原因：行为可拷，建议封禁！
        以免新青年们的思想被此恶心到

**Comment by HLXS-HLXS** [comment]
违规讨论：#5236
举报原因：教唆他人攻击警察，内容无意义，全是贴吧那套恶心人话术

**Comment by Kazuha-hhh** [comment]
违规讨论：#5236
举报原因：营造不良气氛，把开盒违法行为当有趣，不利于现实和绳网秩序

**Comment by keyuanya** [comment]
举报讨论：#5236
举报原因：违法行为

**Comment by baolovey** [comment]
违规讨论：#2548
举报原因：翻墙违法

**Comment by Mrguanboyuan** [comment]
违规文章：#5194
举报原因：发布色情软件

**Reply by fzdqz** [reply]
孩子们到现在我还能用这个贴钓上来鱼:)

**Comment by Liruijia666** [comment]
违规讨论：#5299
举报原因：有问题

**Comment by Lephaethon** [comment]
违规讨论：#5309
举报原因：疑似涩图

**Comment by Lephaethon** [comment]
违规讨论：#4536
举报原因：色图

**Comment by IXfromThorne** [comment]
违规讨论：#4536
举报原因：色图

**Comment by IXfromThorne** [comment]
违规讨论：#5309
举报原因：色图，搬史

**Comment by Loxia1121** [comment]
违规讨论：#5367
举报原因：鬼图

**Comment by March-777th** [comment]
违规讨论：#5429 
举报原因：不好评价<img src="https://bbs-static.miyoushe.com/static/2024/07/31/73346f2b98d45b62da9139d2774fa856_8641114165088315754.png" alt="三月七 告退" width="75" >

**Comment by kk5160** [comment]
违规讨论：#5387
举报原因：搬史带节奏

**Comment by JiuXiat** [comment]
违规讨论：#5387
举报原因：搬史，带节奏

**Comment by JiuXiat** [comment]
违规讨论：#5429
举报原因：恶心别人

**Comment by Noxiil092333** [comment]
违规讨论：#5429
举报原因：迷惑显眼包

**Comment by wshtdgo** [comment]
违规讨论：#5437
举报原因：引战

**Comment by shifcb** [comment]
违规讨论：#5437
举报原因：含有引战言论

**Comment by Junjun280** [comment]
违规讨论：#5437
举报原因：引战

**Comment by shenguojia123** [comment]
违规讨论：#5437
举报原因：引战

**Comment by JiuXiat** [comment]
违规讨论：#5437
举报原因：引战

**Comment by BJ-cmyk** [comment]
违规讨论：#5437
举报原因：引战nt

**Comment by TNCNET** [comment]
违规讨论：#5437
举报原因：引战

**Comment by TrafficLight6** [comment]
违规讨论：#5441
举报原因：引战

**Comment by Elysia-forever1** [comment]
违规讨论：#6071
举报原因：鬼图

**Comment by TianyiLuo0721** [comment]
违规讨论：#6128
举报原因：有人想搞涩涩

**Comment by BeiAne** [comment]
违规讨论：#6019
举报原因：恶意顶贴

**Comment by ghugfyfgyf** [comment]
违规讨论：#6094
举报原因：煌

**Comment by fishqaq123** [comment]
违规讨论：#6131
举报原因：煌

**Comment by qingyangprince** [comment]
违规讨论：#6070
举报原因：太黄了，内容乌苏

**Comment by KAnti-Entropy** [comment]
违规讨论：#2548
举报原因：梯子
